cms/static/js/views/pages/container.js

```diff
@@ -125,6 +125,13 @@ function($, _, Backbone, gettext, BasePage,
                 this.unitOutlineView.render();
 
             }
+            if (this.options.isIframeEmbed) {
+                window.addEventListener('message', (event) => {
+                    if (event.data && event.data.type === 'refreshXBlock') {
```

Comment:
[inform]: In the next PRs we will move away from receiving updated data via iframe reloading.